packages/fast-components-styles-msft/src/utilities/density.ts

```diff
@@ -105,6 +103,41 @@ export function horizontalSpacingNumber(
     );
 }
 
+/**
+ * Returns the width and height for an icon formatted in the provided unit or px by default.
+ *
+ * @param unit The unit of measurement; px by default.
+ */
+export function glyphSize(unit?: string): DesignSystemResolver<string> {
```

Comment:
Can we name this something more obvious, like  `getGlyphPixelWidthAsCSSStringValue`